src/component/renderable/schema/question/Question.ts

```diff
@@ -172,7 +172,7 @@ export default class Question {
   }
 
   public async getAllValuesAsync() {
-    const userActionTaskList = this._parts.map((x) => x.getValuesAsync());
+    const userActionTaskList = this._parts.map(async (x) => { console.log('first', x, await x?.getValuesAsync()); return x?.getValuesAsync() });
```

Comment:
Please remove `console.log()`
```suggestion
    const userActionTaskList = this._parts.map(async (x) =>x?.getValuesAsync());
```